kiliautoml/models/_hugging_face_named_entity_recognition_model.py

```diff
@@ -265,7 +295,10 @@ def _kili_assets_to_hf_ner_dataset(
 
     def _write_asset(self, job_name, labels_to_ids, handler, asset):
         text = self._get_text_from(asset["content"])
-        annotations = asset["labels"][0]["jsonResponse"][job_name]["annotations"]
+        if job_name in asset["labels"][0]["jsonResponse"]:
```

Comment:
We should create an Asset class, and this kind of logic should be managed by a method of the class. But ok for now